code/game/objects/items/weapons/melee/energy.dm

```diff
@@ -168,7 +168,7 @@
 //Can't be activated or deactivated, so no reason to be a subtype of energy
 /obj/item/weapon/melee/energy/blade
 	name = "energy blade"
-	desc = "A concentrated beam of energy in the shape of a blade. Very stylish... and lethal."
```

Comment:
This is definitely worse, it was correct before and you've added incorrect grammar

Reply:
this one is still incorrect, revert